core/templates/dev/head/pages/about-page/about-page.controller.ts

```diff
@@ -12,91 +12,102 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-require('base_components/BaseContentDirective.ts');
+/**
+ * @fileoverview Controllers for the about page.
+ */
+
+require('domain/utilities/UrlInterpolationService.ts');
```

Comment:
This `require()` should be after the `background-banner.directive.ts` `require()` with one empty line between these two. More is explained in https://github.com/oppia/oppia/wiki/Coding-style-guide#webpack

Reply:
Done!